format-delimited/src/main/java/io/cdap/plugin/format/delimited/common/DataTypeDetectorUtils.java

```diff
@@ -102,29 +102,33 @@ public static List<Schema.Field> detectDataTypeOfEachDatasetColumn(
    * @return Array of column names.
    */
   public static String[] setColumnNames(String rawLine, boolean skipHeader,
-                                        boolean enableQuotedValues, String delimiter) {
+                                        boolean enableQuotedValues, String delimiter, String pathField) {
```

Comment:
should not be passing the path field here, as it has nothing to do with data type detection. Adding it to the schema should be done at a different place

Reply:
Removed pathfield